OpenRA.Mods.Common/Traits/Power/CanPowerDown.cs

```diff
@@ -20,9 +20,13 @@ public class CanPowerDownInfo : UpgradableTraitInfo, Requires<PowerInfo>
 		public readonly bool CancelWhenDisabled = false;
 
 		public readonly string IndicatorImage = "poweroff";
-		public readonly string IndicatorSequence = "offline";
+		[SequenceReference("IndicatorImage")] public readonly string IndicatorSequence = "offline";
 
-		public readonly string IndicatorPalette = "chrome";
+		[PaletteReference] public readonly string IndicatorPalette = "chrome";
+
+		public readonly string NotificationType = "Sounds";
```

Comment:
This is a theoretical feature that won't even work due to https://github.com/OpenRA/OpenRA/issues/4690 but I want get this through.